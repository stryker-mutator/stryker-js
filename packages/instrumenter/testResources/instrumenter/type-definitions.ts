/**
 * @see https://github.com/stryker-mutator/stryker-js/issues/2465
 */ 
const flatten = require('lodash/flatten') as typeof import('lodash/flatten');
