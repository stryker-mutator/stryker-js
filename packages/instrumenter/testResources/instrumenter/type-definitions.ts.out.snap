// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`instrumenter integration type declarations should not produce mutants for TS type definitions 1`] = `
"/**
 * @see https://github.com/stryker-mutator/stryker-js/issues/2465
 */
const flatten = (require('lodash/flatten') as typeof import('lodash/flatten'));"
`;
